stylus/tools/mixins.styl

```diff
@@ -87,13 +87,21 @@ reset()
     padding 0
     border  0
 
-global(selector, placeholder)
-    if cssmodules == true
-        :global({selector})
-            @extend {placeholder} // @stylint ignore
-    else
-        {selector}
-            @extend {placeholder} // @stylint ignore
+global(selector, placeholder, type = extend)
```

Comment:
why do we need support for mixins all o a sudden?

Reply:
`cssModulesUtils` & `NativeUtils` are the functions that's create utility classes with their responsive versions, and they allow mixins (actually we have to use mixins because `extend` doesn't work with media queries).

But the legacy function here that create global classes (so utility classes) only allows `extends`.